src/librustc_typeck/diagnostics.rs

```diff
@@ -4096,7 +4096,7 @@ assert_eq!(!Question::No, true);
 "##,
 
 E0609: r##"
-An attempt to access a non-existent field in a struct was performed.
+Attempted to access a non-existent field in a struct was performed.
```

Comment:
Remove "was performed", the sentence doesn't make sense given that it now starts with "Attempted"